spec/models/ability_spec.rb

```diff
@@ -2,36 +2,35 @@
 require 'cancan/matchers'
 
 describe Ability do
-  before(:each) do
-    @member = FactoryGirl.create(:member)
-    @ability = Ability.new(@member)
-  end
+
+  let(:member) { FactoryGirl.create(:member) }
+  let(:ability) { Ability.new(member) }
 
   context "notifications" do
     it 'member can view their own notifications' do
-      @notification = FactoryGirl.create(:notification, :recipient => @member)
-      @ability.should be_able_to(:read, @notification)
+      @notification = FactoryGirl.create(:notification, :recipient => member)
```

Comment:
Not really: I'm talking about [variable scope](http://www.techotopia.com/index.php/Ruby_Variable_Scope). Variables whose names start with "@" are *instance variables* - they live on an object, and can be accessed from any method of that object. But since `@notification` is only accessed from the block in which it's defined, we can replace it with a local variable `notification`, which is only visible from within that block. It's not a huge issue, but it would give our tests slightly better isolation from each other. Concretely, I mean "replace all uses of `@notification` with `notification` and don't introduce any new `let` constructs".